frontends/precinct-scanner/src/app_tally_report_paths.test.tsx

```diff
@@ -1273,3 +1238,178 @@ test('saving to card: polls closed, general election, single precinct', async ()
+    })
+    .get('/precinct-scanner/config/precinct', {
+      body: getPrecinctConfigPrecinct23ResponseBody,
+    })
+    .get('/precinct-scanner/scanner/status', {
+      body: { ...statusNoPaper, ballotsCounted: 2 },
+    });
+  const { card, renderApp } = buildApp();
+  const writeLongObjectMock = jest.spyOn(card, 'writeLongObject');
+  renderApp();
+  await screen.findByText('Insert Your Ballot Below');
+
+  // Pause the polls
+  fetchMock.post('/precinct-scanner/export', GENERAL_SINGLE_PRECINCT_CVRS);
+  const pollWorkerCard = makePollWorkerCard(
+    electionSample2Definition.electionHash
+  );
+  card.insertCard(pollWorkerCard);
+  await screen.findByText('Do you want to close the polls?');
+  userEvent.click(screen.getByText('No'));
+  mockPollsStateChange('polls_paused');
+  userEvent.click(
+    await screen.findByText('Pause Polls for Center Springfield')
+  );
+  await screen.findByText('Polls are paused.');
+  card.removeCard();
+  await advanceTimersAndPromises(1);
+
+  const expectedBallotCounts: Dictionary<BallotCountDetails> = {
+    'undefined,__ALL_PRECINCTS': [1, 1],
+    'undefined,23': [1, 1],
+  };
+  expect(writeLongObjectMock).toHaveBeenCalledTimes(1);
+  expect(writeLongObjectMock).toHaveBeenCalledWith(
+    expect.objectContaining({
+      isLiveMode: false,
+      tallyMachineType: TallySourceMachineType.PRECINCT_SCANNER,
+      totalBallotsScanned: 2,
+      machineId: '0002',
+      timePollsToggled: expect.anything(),
+      timeSaved: expect.anything(),
+      precinctSelection: singlePrecinctSelectionFor('23'),
+      tally: expect.anything(),
+      talliesByPrecinct: expect.anything(),
+      ballotCounts: expectedBallotCounts,
+      isPollsOpen: false,
+    })
+  );
+});
+
+test('TODO printing: polls unpaused', async () => {
+  mockPollsState('polls_paused');
+  fetchMock
+    .get('/precinct-scanner/config/election', {
+      body: electionSample2Definition,
+    })
+    .get('/precinct-scanner/config/precinct', {
+      body: getPrecinctConfigPrecinct23ResponseBody,
+    })
+    .get('/precinct-scanner/scanner/status', {
+      body: { ...statusNoPaper, ballotsCounted: 2 },
+    });
+  const { card, renderApp } = buildApp(true);
+  renderApp();
+  await screen.findByText('Polls Paused');
+
+  // Unpause the polls
+  fetchMock.post('/precinct-scanner/export', GENERAL_SINGLE_PRECINCT_CVRS);
+  const pollWorkerCard = makePollWorkerCard(
+    electionSample2Definition.electionHash
+  );
+  card.insertCard(pollWorkerCard);
+  await screen.findByText('Do you want to open the polls?');
+  userEvent.click(await screen.findByText('Yes, Open the Polls'));
+  mockPollsStateChange('polls_open');
+  await screen.findByText('Polls are open.');
+
+  await expectPrint((printedElement) => {
+    printedElement.getByText('TEST Polls Opened Report for Center Springfield');
+    // TODO: Write testing once polls paused reports are designed vxsuite#2686
+  });
+});
+
+test('saving to card: polls unpaused', async () => {
```

Comment:
AFAICT we don't have any testing right now for the polls paused straight to poll closed transition, probably worth adding a printing and save to card test for that flow.

Reply:
Yeah, I skipped that that one since it's not a use case we expect very much. But, having a test is better :) Will add then merge.